### ISSUE TITLE
Frequency habits not working on NixOS

### ISSUE THREAD

**Issue by arthurSmalle** [wakatara/harsh#28]
**Context**

I installed harsh (0.10.1) from nix packages (unstable branch) today via home-manager.

**Problem**

There seems to be a problem with the frequency habits (or on my system at least).
I used the "ask" command and got this error:
```
Error: A frequency in your habit file has a target value greater than the interval period.
The problem entry to fix is: Too much coffee : 0
```
**Things I tried**

I edited the config and added other frequency habits, this didn't resolve the problem.
The problem got resolved when I removed all frequency habits from my config.

**To Reproduce**

1. Install harsh 0.10.1 on nixos with home-manager
2. run "harsh ask" without editing the config

**Environment:**

- OS: Linux
- OS version: Nixos 24.11
- Harsh version: 0.10.1

**Comment by wakatara** [comment]
Hey there @arthurSmalle 

Thanks for reporting the bug (and trying the app!). Exciting to see people using it with NIxOS.

Ughhhh... I just looked at the code and instantly saw this is why I should really, really not be coding after midnight:  I introduced a bug when allowing intervals (ie. 4 / month etc) which checks if the target is > interval for accounting but does not provide an exception for 0 day habits - ie. tracking but not counting it in your habits. Will correct shortly and release a fix for 0.10.4 tout suite. (actually, embarrassingly, I override it once but forgot in the other place it needs to be.
(target in this case is zero and interval is 1 since we check each day).

This affects all habits that are tracked (ie. 0 days) but not counted in your counts per day.

Will fix as soon as I get some time today and push an 0.10.4 version of the code. Will ping you here to let you know when the new version is live. Should be sometimes this afternoon (Asia time.). 

thanks again for reporting this, 
Daryl.

**Comment by wakatara** [comment]
@arthurSmalle 
This should be fixed now. I pushed version 0.10.4 which you can grab via brew or snap (neither of which I assume you use on Nix :-) ) or via the release and code here: 

https://github.com/wakatara/harsh/releases/tag/v0.10.4

lemme know if you run into any more problems and thanks again for reporting the bug (it's a bit of an embarassing one, really... need to build out the test suite more... 8-/. ). 

ciao !
Daryl.
